test/e2e/performanceprofile/functests/utils/daemonset/daemonset.go

```diff
@@ -43,6 +47,29 @@ func IsRunning(ctx context.Context, cli client.Client, namespace, name string) (
 		}
 		return false, err
 	}
-	testlog.Infof("daemonset %q %q desired %d scheduled %d ready %d", namespace, name, ds.Status.DesiredNumberScheduled, ds.Status.CurrentNumberScheduled, ds.Status.NumberReady)
-	return ds.Status.DesiredNumberScheduled > 0 && ds.Status.DesiredNumberScheduled == ds.Status.NumberReady, nil
+	isRunning := ds.Status.DesiredNumberScheduled > 0 && ds.Status.DesiredNumberScheduled == ds.Status.NumberReady
+	if !isRunning {
```

Comment:
Since we don't need `isRunning` outside the scope of if, we could re-write this as
```
if isRunning := ds.Status.DesiredNumberScheduled > 0 && ds.Status.DesiredNumberScheduled == ds.Status.NumberReady ; !isRunning{
```